docs/understanding-airbyte/high-level-view.md

```diff
@@ -15,33 +15,44 @@ A more concrete diagram can be seen below:
 ```mermaid
 ---
 title: Architecture Overview
-config:
-  theme: neutral
 ---
+%%{init: {"flowchart": {"defaultRenderer": "elk"}} }%%
 flowchart LR
     W[fa:fa-display WebApp/UI]
-    S[fa:fa-server Server/Config API]
+    S[fa:fa-server Config API Server]
     D[(fa:fa-table Config & Jobs)]
-    T(fa:fa-calendar Temporal)
+    L[(fa:fa-server Launcher)]
+    O[(fa:fa-superpowers Orchestrator)]
+    Q[(fa:fa-superpowers Queue)]
+    T(fa:fa-calendar Temporal/Scheduler)
     W2[1..n Airbyte Workers]
+    WL[fa:fa-server Workload API Server]
+    
     W -->|sends API requests| S
     S -->|store data| D
     S -->|create workflow| T
     T -->|launch task| W2
-    W2 -->|return job| T
-    W2 -->|launches| Source
-    W2 -->|launches| Destination
+    W2 -->|return status| T
+    W2 -->|creates job| WL
+    WL -->|queues workload| Q
+    Q  -->|reads from| L
+    L -->|launches| O
+    O -->|launches/reads from| Source
+    O -->|launches/reads from/writes to| Destination
+    O -->|reports status to| WL
 ```
 
 - **Web App/UI** [`airbyte-webapp`, `airbyte-proxy`]: An easy-to-use graphical interface for interacting with the Airbyte API.
-- **Server/Config API** [`airbyte-server`, `airbyte-server-api`]: Handles connection between UI and API. Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API.
+- **Config API Server** [`airbyte-server`, `airbyte-server-api`]: Handles connection between UI and API. Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API.
 - **Database Config & Jobs** [`airbyte-db`]: Stores all the connections information \(credentials, frequency...\).
 - **Temporal Service** [`airbyte-temporal`]: Manages the task queue and workflows.
 - **Worker** [`airbyte-worker`]: The worker connects to a source connector, pulls the data and writes it to a destination.
+- **Workload API** [`airbyte-workload-api-server`]: Manages workloads, Airbyte's internal job abstraction.
+- **Launcher** [`airbyte-workload-launcher`]: Launch workloads.
```

Comment:
```suggestion
- **Launcher** [`airbyte-workload-launcher`]: Launches workloads.
```

Reply:
We are a bit inconsistent on this, but at least we should try to align the paragraph.